Release notes release-2024-09 (#896)
livingdocsIO/documentation: content/operations/releases/release-2024-09.md

### Document Info Panel :gift:

> A new document info panel has been added to show general information about the document. It currently shows information about copies, document statistics and incoming- and outgoing links. 

Reviewer: ```suggestion
A new document info panel has been added to show general information about the document. It currently shows information about copies, document statistics and incoming and outgoing links. 
```

> The copy information from the metadata panel was moved to the document info panel. Additionally the original and copies are shown for copy flows.

Reviewer: ```suggestion
Copy information from the metadata panel has been moved to the Document Info panel. Additionally, the original document and copies of document are shown when using the Document Copy Flows.
```

> Statistics needs to be configured per content type:

Reviewer: ```suggestion
If you want to show document statistics, they have to be configured per content type:
```

```js
// Content Type Config
editor: {
  statistics: {
    characterCount: true,
    componentCount: [
      'title',
      'p',
      'image',
      'image-named-crops'
    ]
  }
}
```

{{< img src="./release-2024-09-document-info-panel.png" alt="Screenshot of a document info panel" >}}

### Document Editing Header and Toolbar :gift:

We improved the layout of the header to better adjust on small screens. Important information like the title will now always be shown and we removed the collaboration bar. Collaborators are now shown directly next to the user avatar.

> In this release, we've streamlined the editor toolbar to enhance its predictability and user-friendliness. We’ve introduced more consistent behavior across actions, panels, and screens, ensuring they no longer interfere with each other.

Reviewer: Could we add that the paths work differently now? Maybe:
```suggestion
In this release, we've streamlined the editor toolbar to enhance its predictability and user-friendliness. We’ve introduced more consistent behavior across actions, panels, and screens, ensuring they no longer interfere with each other.

With this changes we are removing most editor state URLs and redirect URLs `/p/{projectHandle}/articles/{documentId}/*` to `/p/{projectHandle}/document/{documentId}`. As a result, each document has only one URL, `/p/{projectHandle}/document/{documentId}` and the old URLs will redirect to the document URL.
```

### Push Message Proposals :gift:

> It is now possible to add, edit and delete push message proposals. The stored proposals can then be used at a later point to send them. On tabledashboards we now show icons when a proposal exists (black light bulb) and when push message was sent (green bell with a checkmark).

Reviewer: ```suggestion
It is now possible to add, edit and delete push message proposals. The stored proposals can then be retrieved at a later point and editors can interact with them as a plain push message. We also improved Push Messages feedback on TableDashboards, we now show icons when a proposal exists (black light bulb) and when push message was sent (green bell with a checkmark).

This improvements will be shipped with September release for everyone using Push Messages.
```

### Table dashboards: Direct publish without safeguard :gift:

> There are safeguards in place which make sure that it's not possible to direct publish when there are too many changes or document is still unpublished.

Reviewer: ```suggestion
When publishing a document from Table Dashboard cells there are safeguards in place to ensure document quality. This safeguards  make sure that it is not possible to directly publish when there are too many changes or document is still unpublished.
```

> Those safeguards can now be disabled:

Reviewer: ```suggestion
Those safeguards can now be disabled with the following Editor configuration:
```
